utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -360,8 +360,8 @@ int main(int argc, char **argv)
 
         inputTemp = inputu8 + srcDescPtr->offsetInBytes;
         inputSecondTemp = inputu8Second + srcDescPtr->offsetInBytes;
-        inputi8Temp = (Rpp8s *)input + srcDescPtr->offsetInBytes;
-        inputi8SecondTemp = (Rpp8s *)input_second + srcDescPtr->offsetInBytes;
+        inputi8Temp = static_cast<Rpp8s *>(input) + srcDescPtr->offsetInBytes;
```

Comment:
Why are f16/f32 reinterpret_cast but not here for i8? Pls confirm if outputs have been verified with these changes?

Reply:
f16 is not a related data type to U8 so we cant use static_cast to convert u8* to f16*
i8 and u8 related and have the same size and representation in memory, so I used static_cast